### PR TITLE
Clients/Add Button added to main clients page

### PR DESCRIPTION
**Pull request by SamYap-Softwire**
Clients/Add Button added to main clients page

- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've linted via `npm run lint`
    - can run `npm run lint_fix` to try and fix as any mistakes automatically as possible!
- [x] Make sure you've tested via `npm run test`

### CODE REVIEW

File: src/app/clients/ClientsPage.tsx [hunk 1]
```diff
@@ -178,6 +186,11 @@ const ClientsPage: React.FC<Props> = (props) => {
                     <ExpandedClientDetails parcelId={selectedParcelId} />
                 </Suspense>
             </Modal>
+            <CenterComponent>
+                <Button variant="contained" onClick={onClickAddClients}>
```
Comment (DanWilkins2107) [inline src/app/clients/ClientsPage.tsx:190]: Does this need to be separate? Just bc it's a short function anyway and could just be put inline.

File: src/app/clients/ClientsPage.tsx [hunk 2]
```diff
@@ -90,10 +93,15 @@ interface Props {
 }
 
 const ClientsPage: React.FC<Props> = (props) => {
+    const router = useRouter();
     const [selectedParcelId, setSelectedParcelId] = useState<string | null>(null);
 
     const theme = useTheme();
 
+    const onClickAddClients = (): void => {
+        router.push("/clients/add");
```
Comment (TomCusack-Softwire) [inline src/app/clients/ClientsPage.tsx:102]: is this better implemented as a `href` instead of an onclick which takes you to a URL?

**Review by DanWilkins2107 — APPROVED** [review]
Looks good, just the one small edit suggested, feel free to ignore

**Review by Var1377 — CHANGES_REQUESTED** [review]
I agree with @TomCusack-Softwire here, use a Link component instead

**Review by daniel-conlan-softwire — APPROVED** [review]
Looks all good to me for the demo. In future we'll probably change the button position, styling, and how it displays on mobile. But for just now it's all good

**Review by Var1377 — APPROVED** [review]
just add the todo to make sure we eventually change it to a Link

**Review by TomCusack-Softwire — APPROVED** [review]

**Pull request merged**